test/uk/gov/hmrc/merchandiseinbaggage/pact/MibConnectorContractSpec.scala

```diff
@@ -33,13 +33,14 @@ import uk.gov.hmrc.merchandiseinbaggage.{BaseSpecWithApplication, CoreTestData}
 import java.io.File
 import java.time.LocalDate
 
-class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData with MibConfiguration {
+class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData {
```

Comment:
Can you please fix the warnings of naming the boolean params and the statuses. Thanks